.coveragerc

```diff
@@ -1,5 +1,6 @@
 # .coveragerc for edx-platform
 [run]
+branch=true
 data_file = reports/${TEST_SUITE}.coverage
```

Comment:
EIther remove the `${TEST_SUITE}` from this config file, or set the environment variable, and remove the renaming of the file in the action.

Reply:
I think we can remove this variable because it was using with Jenkins. Currently it's not in use, we are using shard names instead. I'll try by removing this variable and see if it works. @nedbat

Reply:
@nedbat After removing this variable, everything is working fine and we are good to merge the changes. Please have a  look then we can go for merge.